tests/rp2040_spinlock/alire.toml

```diff
@@ -12,3 +12,4 @@ atomic = { path='../..' }
 
 [configuration.values]
 atomic.backend = "rp2040_spinlock"
+atomic.RP2040_Spinlock_ID = 21
```

Comment:
Did you mean 31 here?

Reply:
No I am using a value different than the default one on purpose.